src/Symfony/Component/VarDumper/Caster/FrameStub.php

```diff
@@ -18,15 +18,17 @@
  */
 class FrameStub extends EnumStub
 {
-    public $srcContext;
+    const DEFAULT_SRC_CONTEXT = 1;
```

Comment:
I thinking about changing this const to a public static so that the default value can be easily changed